frontend/src/components/PsyListing/NoResultPsyTable.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Text } from '@dataesr/react-dsfr';
+
+const NoResultPsyTable = ({ noResultAction }) => (
+  <Text size="lg">
+    Aucun résultat n&lsquo;a été trouvé, veuillez élargir votre champ de recherche ou
+    <button
```

Comment:
Why not a proper Button ?

Reply:
For style purpose, I don't want "fr-btn" style to have a better integration in the sentence. 
Can be discussed.